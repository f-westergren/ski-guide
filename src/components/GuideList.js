import React from 'react';
import GuideCard from './GuideCard';
import { Grid } from '@material-ui/core';


const GuideList = () => {
  return (
    <Grid container spacing={2}>
      <Grid item xs={12} sm={4}>
        <GuideCard 
          title="Folke Westergren" 
          subtitle="A passionate skier and educator" 
          avatarSrc="https://www.mountaineers.org/blog/wobble-your-way-through-ski-season-finding-fun-as-a-first-time-skier/@@images/image"
          imgSrc="https://content.tui.co.uk/adamtui/2016_11/9_3/3df4164d-2057-4a29-8ee7-a6b90036a8d2/CAN_BAN_F0099WebOriginalCompressed.jpg?i10c=img.resize(width:658);img.crop(width:658%2Cheight:370)"
          description="I have been skiing my entire life. I love skiing. I will ski with you all day and all night!"
        />
      </Grid>
      <Grid item xs={12} sm={4}>
        <GuideCard />
      </Grid>
      <Grid item xs={12} sm={4}>
        <GuideCard />
      </Grid>
    </Grid>
    )
}

export default GuideList;