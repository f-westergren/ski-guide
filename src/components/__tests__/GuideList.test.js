import React from "react";
import { render } from "@testing-library/react";
import GuideList from "../GuideList";

it("renders without crashing", function() {
  render(<GuideList guides={ [{ type: ['ski', 'telemark'] }] } />);
});

it("matches snapshot", function() {
  const { asFragment } = render(<GuideList guides={ [{ type: ['ski', 'telemark'] }] } />);
  expect(asFragment()).toMatchSnapshot();
});
