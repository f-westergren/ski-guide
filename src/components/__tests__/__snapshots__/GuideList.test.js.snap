// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<DocumentFragment>
  <div
    class="container"
  >
    <div
      class="mt-2 card"
    >
      <div
        class="card-body"
      >
        <h6
          class="d-flex justify-content-between"
        >
          <span
            class="text-capitalize"
          />
          <img
            alt="undefined"
            class="thumbnail"
            height="50px"
            src="/placeholder.jpg"
          />
        </h6>
        <p>
          Location: 
        </p>
        <p
          class="text-capitalize"
        >
          ski, telemark
        </p>
        <span>
          No reviews yet
        </span>
      </div>
      <a
        class="stretched-link card-link"
        href="/guides/undefined"
      />
    </div>
  </div>
</DocumentFragment>
`;
